modules/test/playwright/tests/journal-web/journalArticle.spec.ts

```diff
@@ -1791,6 +1792,43 @@ ckeditor5Test(
 				).toBeVisible();
 			}
 		);
+
+		await ckeditor5Test.step('Reset to English', async () => {
+			await journalEditArticlePage.changeLanguage('en_US');
+		});
+
+		await ckeditor5Test.step(
+			'Change article in source editing',
+			async () => {
+				const toolbar =
+					journalEditArticlePage.page.getByLabel('Editor toolbar');
+
+				await toolbar.getByRole('button', {name: 'Source'}).click();
+
+				const textarea = journalEditArticlePage.page.getByLabel(
+					'Source code editing area'
+				);
+
+				await textarea.fill(articleContentSource);
+			}
+		);
+
+		await ckeditor5Test.step('Publish article again', async () => {
+			await journalEditArticlePage.publishArticle(true);
+
+			await expect(page.locator('.alert-success')).toBeVisible();
```

Comment:
I moved this alert assertion outside of `JournalEditArticlePage`, because it didn't work as it was. The reason is that `this.page` reference is scoped to page model of edit article page, and it doesn't fine the alert. Here, we can use generic `page`, that does the trick.

I updated one other use of `publishArticle(true)`, in `layout-content-page-editor-web/fragments.spec.ts`. I couldn't doublecheck if everything was ok with that test, as it is broken for me, failing on an unrelated error.